test/e2e/framework/framework.go

```diff
@@ -678,8 +679,9 @@ func kubectlExec(namespace string, podName, containerName string, args ...string
 		fmt.Sprintf("-c=%v", containerName),
 	}
 	cmdArgs = append(cmdArgs, args...)
+	tc := e2ekubectl.NewTestContext(TestContext.CertDir, TestContext.Host, TestContext.KubeConfig, TestContext.KubeContext, TestContext.KubectlPath)
 
-	cmd := KubectlCmd(cmdArgs...)
+	cmd := tc.KubectlCmd(cmdArgs...)
```

Comment:
The original reason for adding this `NewTestContext` as a pointer receiver for `KubectlCmd`. 

I cannot import `framework/test_context.go` inside of the `k8s.io/kubernetes/test/e2e/framework/kubectl/kubectl_util.go` file because of the circular dependency that it causes, since `framework/framework.go` depends on the `kubectl_util.go` file.